mlflow/store/sftp_artifact_repo.py

```diff
@@ -39,8 +39,14 @@ def __init__(self, artifact_uri, client=None):
             if 'hostname' in user_config:
                 self.config['host'] = user_config['hostname']
 
-            if self.config['username'] is None and 'username' in user_config:
-                self.config['username'] = user_config['username']
+            if self.config['username'] is None and 'user' in user_config:
+                self.config['username'] = user_config['user']
+
+            if self.config['port'] is None:
```

Comment:
this should be `self.config.get('port', None) is None` to avoid `KeyError`.

Reply:
ok, updated.